plugins/importexport/crossref/README

```diff
@@ -18,35 +20,24 @@ complete terms of this license.
 
 System Requirements
 -------------------
-Same requirements as the OJS 2.x core.
+Same requirements as the OJS 3.x core.
 
 Installation
 ------------
 To install the plugin:
  - copy the crossref folder into OJS/plugins/importexport
 
 The export functionality can then be accessed through: 
- - Home > User > Journal Management > Import/Export Data > CrossRef XML Export Plugin
-
-Known Issues/Limitations
----------
-The handling of a DOI suffix is currently hardcoded into the plugin and any changes have to be made 
-by editing the code.  Until such a time when DOI's are part of the OJS core this is seen as the 
-quick and dirty solution.  Once DOIs are handled by the core then this plugin should be updated to 
-reflect as much.
-
-Localization
-------------
-Localized titles are determined from the per-article language setting, rather than the journal locale.  
-As such, localized bibliographic data should be handled properly as per the CrossRef data format 
-guidelines, however, it has not been explicitly tested as of this release.
+ - Tools > Import/Export > CrossRef XML Export Plugin
 
 Contact/Support
 ---------------
-Please email the author for support, bugfixes, or comments.
-Email: <juan@geeky.net>
+For support, bugfixes, or comments please use PKP-LIB GitHub issues: https://github.com/pkp/pkp-lib/issues
 
 Version History
```

Comment:
Likewise, I'd lean towards removing this. It's included in the OJS distro, so no need to version separately, and we're not likely to maintain this history consistently.